sidecar/reloader/main.go

```diff
@@ -160,21 +116,30 @@ func main() {
 		}
 	}()
 
-	// plugin config
-	for _, file := range pluginConfigList {
-		fullFile := filepath.Join(pluginConfigDir, file)
-		if _, err := os.Stat(fullFile); err == nil {
-			log.Printf("add file %s to watcher\n", fullFile)
-			err = watcher.Add(fullFile)
+	//plugin config
+	pluginConfigs, err := ioutil.ReadDir(pluginConfigDir)
+	if err != nil {
+		log.Fatal(err)
+	}
+	for _, pluginConfig := range pluginConfigs {
+		if isBlackFile(pluginConfig.Name()) {
+			continue
+		}
+		filePath := filepath.Join(pluginConfigDir, pluginConfig.Name())
```

Comment:
I think should be check `pluginConfig.Name()`, must match `^emqx.*conf$` or other rule

Reply:
Done